service-commands/src/createUserWithTrackUpload.js

```diff
@@ -0,0 +1,294 @@
+    title: `title_${r6()}`,
+    length: 0,
+    cover_art: null,
+    tags: '',
+    genre: 'SomeGenre',
+    mood: 'Dope',
+    credits_splits: '',
+    create_date: '',
+    release_date: '',
+    file_type: '',
+    description: `description_${r6()}`,
+    license: '',
+    isrc: '',
+    iswc: '',
+    track_segments: []
+  }
+}
+
+/**
+ * Randomly selects url from TRACK_URLS, downloads track file from url to temp local storage, & returns its file path
+ *
+ * @notice this depends on TRACK_URLS pointing to valid urls in S3. Ideally we'd be able to
+ *    randomly select any file from the parent folder.
+ */
+ const getRandomTrackFilePath = async localDirPath => {
+  if (!fs.existsSync(localDirPath)) {
+    fs.mkdirSync(localDirPath)
+  }
+
+  const trackURL = _.sample(TRACK_URLS)
+  const targetFilePath = path.resolve(localDirPath, `${genRandomString(6)}.mp3`)
+
+  const response = await fetch(trackURL)
+  if (!response.ok) {
+    throw new Error(`unexpected response ${response.statusText}`)
+  }
+
+  try {
+    await fs.ensureDir(localDirPath)
+    await streamPipeline(response.body, fs.createWriteStream(targetFilePath))
+
+    console.info(`Wrote track to temp local storage at ${targetFilePath}`)
+  } catch (e) {
+    const errorMsg = `Error with writing track to path ${localDirPath}`
+    console.error(errorMsg, e)
+    throw new Error(`${errorMsg}: ${e.message}`)
+  }
+
+  // Return full file path
+  return targetFilePath
+}
+
+// =~*=~*=~*=~*=~*=~*=~*=~*=~*=~* END UTIL =~*=~*=~*=~*=~*=~*=~*=~*=~*=~*
+const configureAndInitLibs = async () => {
+  const audiusLibsConfig = {
+    ethWeb3Config: AudiusLibs.configEthWeb3(
+      ETH_TOKEN_ADDRESS,
+      ETH_REGISTRY_ADDRESS,
+      ETH_PROVIDER_ENDPOINT,
+      ETH_OWNER_WALLET
+    ),
+    web3Config: AudiusLibs.configInternalWeb3(
+      DATA_CONTRACTS_REGISTRY_ADDRESS,
+      DATA_CONTRACTS_PROVIDER_ENDPOINT
+    ),
+    creatorNodeConfig: AudiusLibs.configCreatorNode(USER_METADATA_ENDPOINT),
+    discoveryProviderConfig: AudiusLibs.configDiscoveryProvider(new Set([DISCOVERY_NODE_ENDPOINT])),
+    identityServiceConfig: AudiusLibs.configIdentityService(IDENTITY_SERVICE_ENDPOINT),
+    isServer: true,
+    isDebug: true
+  }
+
+  const audiusLibs = new AudiusLibs(audiusLibsConfig)
+
+  try {
+    await audiusLibs.init()
+  } catch (e) {
+    console.error('Couldn\'t init libs', e)
+    throw e
+  }
+
+  // Clears local storage in case there is any pre-existing user state
+  audiusLibs.userStateManager.clearUser()
+
+  AudiusLibsInstance = audiusLibs
+
+  return audiusLibs
+}
+
+async function createUser () {
+  userMetadata = getRandomUser()
+  console.log('Creating user with metadata:\n', userMetadata)
+  const { email, password, profilePictureFile, coverPhotoFile } = userMetadata
+  const signUpResp = await AudiusLibsInstance.Account.signUp(
+    email, password, userMetadata/*, profilePictureFile, coverPhotoFile */
+  )
+
+  wait(5000) // wait 5 sec indexing
```

Comment:
do we need this 5 sec check? we don't need discovery to move to the next phase which is upgrade to creator